.github/workflows/pull-requests.yml

```diff
@@ -0,0 +1,22 @@
+name: Pull Requests
+
+on: [pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x]
```

Comment:
Currently runs almost all things that are in package.json. Didn't know if we should also add npm run version or not.